tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/RbValue.java

```diff
@@ -17,20 +18,32 @@
  */
 public final class RbValue {
     private final ImmutableList<String> elements;
+    private final int elementsPerLine;
 
     /** Returns a resource bundle value of the given elements. */
     public static RbValue of(String... elements) {
-        return of(Arrays.asList(elements));
+        return new RbValue(ImmutableList.copyOf(elements), 1);
     }
 
     /** Returns a resource bundle value of the given elements. */
     public static RbValue of(Iterable<String> elements) {
-        return new RbValue(elements);
+        return new RbValue(ImmutableList.copyOf(elements), 1);
     }
 
-    private RbValue(Iterable<String> elements) {
-        this.elements = ImmutableList.copyOf(elements);
-        checkArgument(!this.elements.isEmpty(), "Resource bundle values cannot be empty");
+    /** Returns a resource bundle value of the given elements by consuming the given stream. */
```

Comment:
Basic improvements, mainly to support N-per-line formatting to mimic existing manual formatting of:
https://github.com/unicode-org/icu/blob/master/icu4j/tools/misc/src/com/ibm/icu/dev/tool/locale/LocaleDistanceBuilder.java#L892